index.js

```diff
@@ -29,31 +31,53 @@ const run = config => {
     });
 };
 
-const checkDirectoryStatus = dest => {
-    return fse.stat(dest)
-        .then(statResp => {
-            if (!statResp.isDirectory) {
-                return fse.mkdir(dest);
-            }
-            return Promise.resolve();
-        })
-        .catch(() => fse.mkdir(dest));
-};
-
-const copyDirectory = fileConfig => {
-    return checkDirectoryStatus(fileConfig.dest).then(() => {
-        return fse.readdir(fileConfig.src)
-            .then(files => {
-                const filePromises = files.map(file => copyFile(fileConfig, file));
-                return Promise.all(filePromises);
-            });
+/**
+ * 
+ * @param {String} srcPath the full src path from the config
+ * This function finds the base path which is the path before any glob syntax is used
+ * This helps parse the file path in the copyFile method
+ * @returns {String} base path (the path before any glob syntx is used)
+ */
+function getSourceDir(srcPath) {
+    const srcPaths = srcPath.split('/');
```

Comment:
The '/' path separator is only valid for Unix-based machines. Windows uses a different path separator.  Checkout path.sep and/or path.parse()